internal/conf/configuration.go

```diff
@@ -271,6 +271,7 @@ type TwilioProviderConfiguration struct {
 	AccountSid        string `json:"account_sid" split_words:"true"`
 	AuthToken         string `json:"auth_token" split_words:"true"`
 	MessageServiceSid string `json:"message_service_sid" split_words:"true"`
+	ContentSid        string `json:"content_sid" split_words:"true"`
```

Comment:
does the `ContentSid` env var only affect whatsapp? does twilio's API allow specifying a `ContentSid` for programming messaging with SMS?

Reply:
Twilio's API [allows for specifying a `ContentSid` with programmable messaging](https://www.twilio.com/docs/content/send-templates-created-with-the-content-editor). Unlike with WhatsApp, the `Body` pararmeter [will remain supported](https://www.twilio.com/docs/sms/api/message-resource#create-a-message-resource) on SMS though